pkg/asset/installconfig/vsphere/validation.go

```diff
@@ -20,6 +20,10 @@ func Validate(ic *types.InstallConfig) error {
 		return errors.New(field.Required(field.NewPath("platform", "vsphere"), "vSphere validation requires a vSphere platform configuration").Error())
 	}
 
+	if err := validateCPUConfig(ic); err != nil {
+		return err
+	}
```

Comment:
All static validation--that is validation of user input where it is not necessary to call the vsphere API--should go in types, specifically: https://github.com/openshift/installer/blob/master/pkg/types/vsphere/validation/machinepool.go